hazelcast/src/main/java/com/hazelcast/map/impl/operation/MapReplicationStateHolder.java

```diff
@@ -109,6 +111,14 @@ void applyState() {
                 recordStore.reset();
                 recordStore.setPreMigrationLoadedStatus(loaded.get(mapName));
 
+                MapContainer mapContainer = recordStore.getMapContainer();
```

Comment:
Ah, you mean `PostJoinMapOperation` replicates all indexes (global and per-partition) when joining and `AddIndexOperation` adds them when the cluster is stable (but only on partition owner). This class replicates them after they've been added on the partition owner to the backup replicas?